core/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirectorFactory.java

```diff
@@ -94,14 +95,14 @@ public void setTrackingWorkingSolution(boolean trackingWorkingSolution) {
 
     @Override
     public InnerScoreDirector<Solution_, Score_> buildScoreDirector() {
-        return buildScoreDirector(true, true);
+        return buildScoreDirector(false, ConstraintMatchPolicy.DISABLED);
```

Comment:
Should we set `ENABLED_WITHOUT_JUSTIFICATIONS` or `ENABLED` instead?

Reply:
The method is only used in tests. I will remove it entirely.

Reply:
I understand the reason now after reading the comments in the `InnerScoreDirectorFactory`

Reply:
Sounds good